store/tikv/coprocessor.go

```diff
@@ -326,7 +324,10 @@ func (it *copIterator) work() {
 		task.status = taskRunning
 		it.mu.Unlock()
 		bo := NewBackoffer(copNextMaxBackoff, context.Background())
+		startTime := time.Now()
 		resp, err := it.handleTask(bo, task)
+		coprocessorHistogram.Observe(time.Since(startTime).Seconds())
+		backoffHistogram.Observe(float64(bo.totalSleep) / 1000)
```

Comment:
What does 'pure coprocessor time' mean exactly?
We already have _tidb_tikvclient_request_seconds{type="cop"}_ for the request round trip, and also _tikv_coprocessor_request_duration_seconds_bucket_ on server side. It may not be necessary to use subtraction.